src/Symfony/Component/HttpKernel/KernelInterface.php

```diff
@@ -127,6 +123,14 @@ public function getCacheDir(): string;
      */
     public function getBuildDir(): string;
 
+    /**
+     * Returns the share directory.
+     *
+     * This directory should be used to store data that is shared between all front-end servers.
+     * This typically fits application caches.
+     */
+    public function getShareDir(): string;
```

Comment:
should this be nullable for cases where our infrastructure does not actually support having a filesystem path that is shared ?
How are we supposed to configure the kernel on such infrastructure (for instance on Heroku) ?